### PR TITLE
Add CSS AutoPrefixer

### PR DESCRIPTION
**Pull request by AntonSermyazhko**

### CODE REVIEW

**Review by lukyanovas — DISMISSED** [review]
LGTM

**Comment by AlekseyMartynov** [comment]
Now when we have autoprefixer, should we clean manually coded prefixes?

https://github.com/DevExpress/DevExtreme/blob/2ba8f526363abb2c6717a119c967ccc445582cd4/styles/widgets/ui.less#L58-L61

**Comment by AntonSermyazhko** [comment]
Yes, we should. Adding prefixes is an autoprefixer job.

**Review by lukyanovas — DISMISSED** [review]

**Comment by AlekseyMartynov** [comment]
@lukyanovas wait wait, it will get even better!

File: package.json [hunk 1]
```diff
@@ -104,5 +105,11 @@
     "dev": "gulp dev",
     "test-env": "node testing/launch"
   },
+  "browserslist": [
+    "last 2 versions",
+    "ios > 7",
```
Comment (DokaRus) [inline package.json]: Why not `ios > 8`? 8.x => ~0.45%
https://data.apteligent.com/ios/

Comment (nightskylark) [reply package.json]: I have seen the separate file `browserslist` in some projects. Is it possible to extract such file here?

Comment (AntonSermyazhko) [reply package.json]: As for me, not a problem to increase the version of iOS to 9+.
Think that a separate file will be superfluous.

Comment (AlekseyMartynov) [reply package.json]: @nightskylark 
package.json is easier to read.
.browserslistrc has comments, etc, and it's [not supported automatically](https://github.com/less/less-plugin-autoprefix/issues/31#issuecomment-405000747) by the plugin

**Review by AlekseyMartynov — APPROVED** [review]

**Pull request merged**